src/main/target/OMNIBUSF4FW/target.h

```diff
@@ -209,7 +213,7 @@
 
 #define TARGET_IO_PORTA (0xffff & ~(BIT(14)|BIT(13)))
 #define TARGET_IO_PORTB (0xffff & ~(BIT(2)))
-#define TARGET_IO_PORTC (0xffff & ~(BIT(15)))
+#define TARGET_IO_PORTC (0xffff & ~(BIT(16)))
```

Comment:
Ports only have 16 pins, so this won't work (they are also 0 based).

Reply:
@mikeller I have tried it and I think that if it works, without the modification, this error jumps, with the perfect modification.
./src/main/pg/max7456.c:38:21: error: 'defio_error_PC15_is_not_supported_on_TARGET' undeclared (first use in this function)
     config->csTag = IO_TAG(MAX7456_SPI_CS_PIN);

Reply:
`TARGET_IO_PORTC` defines pins that are available on PORTC - each usable pin is set to one. 
`(0xffff & ~(BIT(15)))` means all pins except PC15. If PC15 is available, simply use `#define TARGET_IO_PORTC 0xffff`

Reply:
@ledvinap great!thanks for the explanation,im going to correct it